cmd/notation/verify.go

```diff
@@ -101,10 +99,15 @@ func runVerify(command *cobra.Command, opts *verifyOpts) error {
 		MaxSignatureAttempts: math.MaxInt64,
 	}
 
-	// core process
+	// core verify process.
 	_, outcomes, err := notation.Verify(ctx, verifier, sigRepo, verifyOpts)
 	if err != nil {
-		logger.Error(err)
+		var errorVerificationFailed *notation.ErrorVerificationFailed
+		if !errors.As(err, &errorVerificationFailed) {
+			// print out the error if it's not the general
+			// ErrorVerificationFailed
+			fmt.Println("Verification error:", err)
```

Comment:
In this case, should we just return the error?

Reply:
It is weird to have something like
```
Verification error: somewhat network error
Error: signature verification failed for all the signatures associated with bla bla bla
```

Reply:
Based on our spec, the final error is returned at line 114. This fmt here is to provide more details on what actually failed the verification.